config/config/sylius_template_events.php

```diff
@@ -52,7 +60,15 @@
                     'visa_mobile' => [
                         'template' => '@CommerceWeaversSyliusTpayPlugin/shop/cart/complete/_visaMobile.html.twig',
                         'priority' => 5,
-                    ]
+                    ],
+                    'google_pay' => [
+                        'template' => '@CommerceWeaversSyliusTpayPlugin/shop/cart/complete/_google_pay_regulations.html.twig',
```

Comment:
I'm not sure what to do with it, but I'm not convinced about the file naming of our events. We are using both snake case and camel case. But the problem is existing without this PR